next.config.js

```diff
@@ -12,7 +12,7 @@ const isDev = process.env.NODE_ENV !== 'production'
 const ENABLE_CSP_HEADER = true;
 const FRAME_SRC_HOSTS = ['https://*.walletconnect.com', 'https://*.walletconnect.org','https://*.solflare.com'];
 const STYLE_SRC_HOSTS = ['https://*.googleapis.com']
-const IMG_SRC_HOSTS = ['https://*.walletconnect.com'];
+const IMG_SRC_HOSTS = ['https://*.walletconnect.com', 'https://*.githubusercontent.com'];
```

Comment:
should we allow 
`NEXT_REGISTRY_URL`?

Reply:
or alternatively force that this CORS stuff happens at build time

Reply:
We only support github-based remote registries atm so when would NEXT_PUBLIC_REGISTRY_URL have a different URL than this?

Reply:
> CORS stuff
CSP !== CORS :)

Reply:
> We only support github-based remote registries atm so when would NEXT_PUBLIC_REGISTRY_URL have a different URL than this?
fair enough